src/color/alert.json

```diff
@@ -2,172 +2,87 @@
-          "deprecated": false
-        },
-        "onDark": {
-          "value": "{color.state.error.100.value}",
-          "public": false,
-          "onDark": true,
-          "usage": "Error color on dark backgrounds",
-          "wcag": "AAA",
           "deprecated": true,
-          "reference": "n/a",
-          "comment": "Deprecated token",
-          "version": "3.12.0"
+          "version": "4.5.0",
+          "reference": "n/a"
         }
       },
       "success": {
-        "onLight": {
-          "value": "{color.state.success.500.value}",
-          "public": false,
-          "onLight": true,
-          "usage": "Success color on light backgrounds",
-          "wcag": "AAA",
-          "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "color-alert-success-default",
-          "version": "3.15.0"
-        },
         "default": {
           "value": "{color.state.success.500.value}",
-          "public": true,
+          "public": false,
           "default": true,
           "usage": "Success color on light backgrounds",
           "wcag": "AAA",
-          "deprecated": false
-        },
-        "onDark": {
-          "value": "{color.state.success.100.value}",
-          "public": false,
-          "onDark": true,
-          "usage": "Success color on dark backgrounds",
-          "wcag": "AAA",
           "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "n/a",
-          "version": "3.12.0"
+          "version": "4.5.0",
+          "reference": "n/a"
         }
       },
       "advisory": {
-        "onLight": {
-          "value": "{color.brand.goldcoast.100.value}",
-          "public": false,
-          "onLight": true,
-          "usage": "Travel advisory or system maintenance color on light backgrounds",
-          "wcag": "AAA",
-          "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "color-alert-advisory-default",
-          "version": "3.15.0"
-        },
         "default": {
           "value": "{color.brand.goldcoast.100.value}",
-          "public": true,
+          "public": false,
           "default": true,
           "usage": "Travel advisory or system maintenance color on light backgrounds",
           "wcag": "AAA",
-          "deprecated": false
-        },
-        "onDark": {
-          "value": "{color.brand.goldcoast.100.value}",
-          "public": false,
-          "onDark": true,
-          "usage": "Travel advisory or system maintenance color on dark backgrounds",
-          "wcag": "AAA",
           "deprecated": true,
-          "comment": "Deprecated token",
-          "reference": "n/a",
-          "version": "3.12.0"
+          "version": "4.5.0",
+          "reference": "n/a"
         }
       },
       "bkgSuccess": {
         "default": {
           "value": "DDF6E8",
-          "public": true,
-          "onLight": true,
+          "public": false,
```

Comment:
Why 'false'?

Reply:
Changed to `false` because I saw other `deprecated` tokens with this change in the past.